english/light-fire/sprinkler/basement-swim/dried-chocolate/chocolate-cocoon/chocolate-cocoon.md

```diff
@@ -0,0 +1,9 @@
+As you wait for the chocolate to dry, you quickly fall asleep and start dreaming of being a kid again. You dream
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.